src/backends/github/API.js

```diff
@@ -64,7 +68,9 @@ export default class API {
     return fetch(url, { ...options, headers }).then((response) => {
       responseStatus = response.status;
       const contentType = response.headers.get("Content-Type");
-      if (contentType && contentType.match(/json/)) {
+      if (url.indexOf('/collaborators/') >= 0) {
+        return responseStatus;
+      } else if (contentType && contentType.match(/json/)) {
```

Comment:
@Benaiah Would `isCollaborator` then need to follow promise syntax and utilize `resolve()` etc.? I'm of the understanding that without doing that `.then((response)...` will still work based on what I have now.